src/public/js/at-sce.util.js

```diff
@@ -42,6 +42,81 @@ $(function () {
 });
 
 /* Calculator Functions */
+function SumFixedCost() {
+  var rent = document.getElementById("rent").value;
+  var transport = document.getElementById("transport").value;
+  var telephone = document.getElementById("telephone").value;
+  var feeding = document.getElementById("feeding").value;
+  var others = document.getElementById("others").value;
+  var totalFixedCost =
+    parseFloat(rent) +
+    parseFloat(transport) +
+    parseFloat(telephone) +
+    parseFloat(feeding) +
+    parseFloat(others);
+  return totalFixedCost;
+}
+
+function CostPerDay() {
```

Comment:
Try to name your function with its purpose for readability, in here I see costPerDay but it could be easily switched to calculateCostPerDay() which makes is more readable and helps another developer to know what the function is doing without even checking the inside code

Reply:
`function calcTotalCostPerDay() `